mailet/standard/src/main/java/org/apache/james/transport/mailets/MailAttributesToMimeHeaders.java

```diff
@@ -54,20 +56,10 @@
 
     @Override
     public void init() throws MessagingException {
-        String simplemappings = getInitParameter("simplemapping");
-        Builder<String, String> mappingsBuilder = ImmutableMap.builder();
-        if (simplemappings != null) {
-            for (String mapping : Splitter.on(',').split(simplemappings)) {
-                List<String> pair = Splitter.on(';').trimResults().splitToList(mapping);
-                if (pair.size() != 2) {
-                    throw new MessagingException(CONFIGURATION_ERROR_MESSAGE);
-                }
-                mappingsBuilder.put(pair.get(0), pair.get(1));
-            }
-        } else {
-            throw new MessagingException(CONFIGURATION_ERROR_MESSAGE);
-        }
-        mappings = mappingsBuilder.build();
+        String simpleMappings = getInitParameter("simplemapping");
+        Preconditions.checkArgument(!Strings.isNullOrEmpty(simpleMappings), "simplemapping is required");
```

Comment:
The precondition using to checking for all arguments and it's called at the first line of function, if not, using the normal checking instead of Precondition. WDYT?